pandas/tests/tseries/data/dateoffset_0_19_2.pickle

```diff
@@ -0,0 +1,187 @@
+(dp0
+S'YearBegin'
+p1
+ccopy_reg
+_reconstructor
```

Comment:
not what i asked for at all

pls use the generate_legacy_data script
we have removed all of these ad hoc pickle things
we r not adding them back

Reply:
You asked me to run a script that doesn't work, then to troubleshoot it until it does.  I found existing tests and followed their example.  Please cut me some darn slack.

> we have removed all of these ad hoc pickle things

`test_pickle_v0_15_2` and `test_pickle_compat_0_14_1` might lead a naive reader (such as myself) to suspect otherwise.

> you can use pandas versions to do that ONLY in that script to do that. 

This sentence confuses me.

> If you make changes [to the generation script] then they should be checked in as part of this PR.

I have no interest in making changes to the non-functional generation script.

Reply:
> I have no interest in making changes to the non-functional generation script.

well it works fine according to the instructions I gave and has been used for quite a while.

> test_pickle_v0_15_2 and test_pickle_compat_0_14_1 might lead a naive reader

well if you look at what exists, the vast majority are in an organized format. The remaining ones should actually be converted to this format.
```
(pandas) bash-3.2$ find pandas -name '*pickle*'
pandas/compat/__pycache__/pickle_compat.cpython-36.pyc
pandas/compat/pickle_compat.py
pandas/io/__pycache__/pickle.cpython-36.pyc
pandas/io/pickle.py
pandas/tests/indexes/data/mindex_073.pickle
pandas/tests/indexes/data/multiindex_v1.pickle
pandas/tests/io/__pycache__/test_pickle.cpython-36-PYTEST.pyc
pandas/tests/io/data/categorical_0_14_1.pickle
pandas/tests/io/data/categorical_0_15_2.pickle
pandas/tests/io/data/legacy_pickle
pandas/tests/io/data/legacy_pickle/0.10.1/AMD64_windows_2.7.3.pickle
pandas/tests/io/data/legacy_pickle/0.10.1/x86_64_linux_2.7.3.pickle
pandas/tests/io/data/legacy_pickle/0.11.0/0.11.0_x86_64_linux_3.3.0.pickle
pandas/tests/io/data/legacy_pickle/0.11.0/x86_64_linux_2.7.3.pickle
pandas/tests/io/data/legacy_pickle/0.11.0/x86_64_linux_3.3.0.pickle
pandas/tests/io/data/legacy_pickle/0.12.0/0.12.0_AMD64_windows_2.7.3.pickle
pandas/tests/io/data/legacy_pickle/0.12.0/0.12.0_x86_64_linux_2.7.3.pickle
pandas/tests/io/data/legacy_pickle/0.13.0/0.13.0_AMD64_windows_2.7.3.pickle
pandas/tests/io/data/legacy_pickle/0.13.0/0.13.0_i686_linux_2.6.5.pickle
pandas/tests/io/data/legacy_pickle/0.13.0/0.13.0_i686_linux_2.7.3.pickle
pandas/tests/io/data/legacy_pickle/0.13.0/0.13.0_i686_linux_3.2.3.pickle
pandas/tests/io/data/legacy_pickle/0.13.0/0.13.0_x86_64_darwin_2.7.5.pickle
pandas/tests/io/data/legacy_pickle/0.13.0/0.13.0_x86_64_darwin_2.7.6.pickle
pandas/tests/io/data/legacy_pickle/0.13.0/0.13.0_x86_64_linux_2.7.3.pickle
pandas/tests/io/data/legacy_pickle/0.13.0/0.13.0_x86_64_linux_2.7.8.pickle
pandas/tests/io/data/legacy_pickle/0.13.0/0.13.0_x86_64_linux_3.3.0.pickle
pandas/tests/io/data/legacy_pickle/0.14.0/0.14.0_x86_64_darwin_2.7.6.pickle
pandas/tests/io/data/legacy_pickle/0.14.0/0.14.0_x86_64_linux_2.7.8.pickle
pandas/tests/io/data/legacy_pickle/0.14.1/0.14.1_x86_64_darwin_2.7.12.pickle
pandas/tests/io/data/legacy_pickle/0.14.1/0.14.1_x86_64_linux_2.7.8.pickle
pandas/tests/io/data/legacy_pickle/0.15.0/0.15.0_x86_64_darwin_2.7.12.pickle
pandas/tests/io/data/legacy_pickle/0.15.0/0.15.0_x86_64_linux_2.7.8.pickle
pandas/tests/io/data/legacy_pickle/0.15.2/0.15.2_x86_64_darwin_2.7.9.pickle
pandas/tests/io/data/legacy_pickle/0.16.0/0.16.0_x86_64_darwin_2.7.9.pickle
pandas/tests/io/data/legacy_pickle/0.16.2/0.16.2_AMD64_windows_2.7.10.pickle
pandas/tests/io/data/legacy_pickle/0.16.2/0.16.2_AMD64_windows_3.4.3.pickle
pandas/tests/io/data/legacy_pickle/0.16.2/0.16.2_x86_64_darwin_2.7.10.pickle
pandas/tests/io/data/legacy_pickle/0.16.2/0.16.2_x86_64_darwin_2.7.9.pickle
pandas/tests/io/data/legacy_pickle/0.16.2/0.16.2_x86_64_darwin_3.4.3.pickle
pandas/tests/io/data/legacy_pickle/0.16.2/0.16.2_x86_64_linux_2.7.10.pickle
pandas/tests/io/data/legacy_pickle/0.16.2/0.16.2_x86_64_linux_3.4.3.pickle
pandas/tests/io/data/legacy_pickle/0.17.0/0.17.0_AMD64_windows_2.7.11.pickle
pandas/tests/io/data/legacy_pickle/0.17.0/0.17.0_AMD64_windows_3.4.4.pickle
pandas/tests/io/data/legacy_pickle/0.17.0/0.17.0_x86_64_darwin_2.7.11.pickle
pandas/tests/io/data/legacy_pickle/0.17.0/0.17.0_x86_64_darwin_3.4.4.pickle
pandas/tests/io/data/legacy_pickle/0.17.0/0.17.0_x86_64_linux_2.7.11.pickle
pandas/tests/io/data/legacy_pickle/0.17.0/0.17.0_x86_64_linux_3.4.4.pickle
pandas/tests/io/data/legacy_pickle/0.17.0/0.17.1_AMD64_windows_2.7.11.pickle
pandas/tests/io/data/legacy_pickle/0.17.1/0.17.1_AMD64_windows_2.7.11.pickle
pandas/tests/io/data/legacy_pickle/0.17.1/0.17.1_x86_64_darwin_2.7.11.pickle
pandas/tests/io/data/legacy_pickle/0.18.0/0.18.0_AMD64_windows_2.7.11.pickle
pandas/tests/io/data/legacy_pickle/0.18.0/0.18.0_AMD64_windows_3.5.1.pickle
pandas/tests/io/data/legacy_pickle/0.18.0/0.18.0_x86_64_darwin_2.7.11.pickle
pandas/tests/io/data/legacy_pickle/0.18.0/0.18.0_x86_64_darwin_3.5.1.pickle
pandas/tests/io/data/legacy_pickle/0.18.1/0.18.1_x86_64_darwin_2.7.12.pickle
pandas/tests/io/data/legacy_pickle/0.18.1/0.18.1_x86_64_darwin_3.5.2.pickle
pandas/tests/io/data/legacy_pickle/0.19.2/0.19.2_x86_64_darwin_2.7.12.pickle
pandas/tests/io/data/legacy_pickle/0.19.2/0.19.2_x86_64_darwin_3.6.1.pickle
pandas/tests/io/test_pickle.py
pandas/tests/tseries/data/cday-0.14.1.pickle
pandas/tests/tseries/data/dateoffset_0_15_2.pickle
```

As you know pandas is a very large library. If things do not have process & procedure around them, then future generations will have pain. We have worked pretty hard (and you are contributing to this), to reduce technical debt.